frontend/api/members.graphql

```diff
@@ -15,6 +15,12 @@ query MeHeader {
         nameEn
       }
     }
+    customAuthorOptions {
```

Comment:
This is fine since it's in line with what we generally do when we add new fields to the backend, but long term it makes our use of GraphQL somewhat pointless. The goal of GraphQL is to only fetch exactly the date that the current view needs. We should consider splitting all of these large queries up at some point, or transition away from GraphQL.

Reply:
+1 on transitioning away from graphql